### ISSUE TITLE
TODO for paper #2: DPGP clustering analysis 

### ISSUE THREAD

**Issue by annashcherbina** [kundajelab/DMSO#11]
Same as we did for hets. Need to run the clustering analysis for DMSO as well.

**Comment by annashcherbina** [comment]
DPGP clustering analysis for peaks: 



Order is: 
Control earlyG1 
Control lateG1 
Control SG2M 
DMSO earlyG1 
DMSO lateG1 
DMSO SG2M 

![dpgp_diff_peaks_posterior_similarity_matrix_heatmap](https://user-images.githubusercontent.com/5261545/32348328-a6f8b6da-bfd0-11e7-8d63-6c1372ca4c45.png)

![dpgp_diff_peaks_cluster_sizes](https://user-images.githubusercontent.com/5261545/32348331-aace8f28-bfd0-11e7-8276-b07d4d1a6871.png)


![dpgp_diff_peaks_gene_expression_fig_1](https://user-images.githubusercontent.com/5261545/32348337-ae21febc-bfd0-11e7-8ad5-f334a3142f38.png)


![dpgp_diff_peaks_gene_expression_fig_2](https://user-images.githubusercontent.com/5261545/32348340-b1162030-bfd0-11e7-995f-7dbd486ca8ba.png)


DPGP clustering analysis for genes: 

![dpgp_diff_genes_posterior_similarity_matrix_heatmap](https://user-images.githubusercontent.com/5261545/32348275-7b22e6d4-bfd0-11e7-9882-550343086281.png)


![dpgp_diff_genes_cluster_sizes](https://user-images.githubusercontent.com/5261545/32348279-7f907808-bfd0-11e7-8fa9-c7f50a7d2fd9.png)


Order is: 
Control earlyG1 
Control lateG1 
Control SG2M 
DMSO earlyG1 
DMSO lateG1 
DMSO SG2M 

![dpgp_diff_genes_gene_expression_fig_1](https://user-images.githubusercontent.com/5261545/32348302-96c61d52-bfd0-11e7-869a-4640373bce41.png)
![dpgp_diff_genes_gene_expression_fig_2](https://user-images.githubusercontent.com/5261545/32348304-98948a56-bfd0-11e7-9c02-3c843b646b5d.png)
![dpgp_diff_genes_gene_expression_fig_3](https://user-images.githubusercontent.com/5261545/32348308-9b39ead0-bfd0-11e7-808e-999aefe852f9.png)
![dpgp_diff_genes_gene_expression_fig_4](https://user-images.githubusercontent.com/5261545/32348310-9d905b8e-bfd0-11e7-88ff-ac459a343f97.png)
![dpgp_diff_genes_gene_expression_fig_5](https://user-images.githubusercontent.com/5261545/32348316-a07412fa-bfd0-11e7-8d3f-b4e86ae57650.png)

**Comment by annashcherbina** [comment]
More organized version of clustering figures: 
https://docs.google.com/presentation/d/11tsBKPbJ3LMEk1kYkqe2A_xl6FJFoUmooEcWJDWRf5s/edit#slide=id.g2a8f18cee3_0_560

**Comment by akundaje** [comment]
Anna,

It doesnt make sense to use DPGP clustering across different disconnected
time series. You can use it separately for control and separately for
treatment but not as a concatenation. DPGP has an explicit notion of time.
The control and treatment are not a continuous time series.

-Anshul.

**Comment by annashcherbina** [comment]
Should we just stick with k-means clustering for DMSO analysis then? 
Or should DPGP be done on the fold change of DMSO vs Control over time? 
Really we mostly care about the DMSO vs Control effect, not the time effect within the DMSO samples or controls separately.

**Comment by akundaje** [comment]
Kmeans is fine

**Comment by annashcherbina** [comment]
I think Sundari wants the DPGP -- I already sent her those figures, so is it valid if I re-run it on the fold change? I already have kmeans results as backup if needed.

**Comment by akundaje** [comment]
Dpgp cannot be used for disparate time series because it assumes contiguous
temporal order across the samples as you provide them. So you should switch
to kmeans.

**Comment by annashcherbina** [comment]
Yes I know, that's why I suggested the fold change of DMSO vs control. 
So the three timepoints would be 
DMSO / Control in earlyG1 
DMSO / Control in lateG1
DMSO / Control in SG2M

That is a single time series, so why would that be wrong? 

We basically get the same results with the DPGP & the kmeans though: https://github.com/kundajelab/DMSO/issues/7 so I can go with those, but I think the DPGP visualizations are better.

**Comment by akundaje** [comment]
Ah I see. I didn't realize that's what you meant. I thought you meant fold
change relative to expected background for ATAC. Sure. That would be fine.
